Rules/CommonScripts/ScoreboardRender.as

```diff
@@ -1,5 +1,6 @@
 #include "ScoreboardCommon.as";
 #include "Accolades.as";
+#include "RunnerHead.as";
```

Comment:
This include still needs to be removed before this PR can be merged I'm afraid!

Reply:
Oh I thought you would remove it when merging. I’ll do that ASAP.

Reply:
Just remembered that it's needed to get the heads pack index on line 154. Is the reason you want it removed since you thought it was not needed?

Reply:
No it's that it'll pull in extra hooks and a heap of extra code for what is essentially getting one string (the heads sheet) and one integer (the head frame) 😅

A solution might be storing these in a straightforward blob property (so mods can also use the functionality easily) and displaying a skull (or nothing) when the player's blob is dead.